pkg/cmd/server/origin/auth.go

```diff
@@ -459,15 +460,16 @@ func (c *AuthConfig) getPasswordAuthenticator(identityProvider configapi.Identit
 
 		opts := ldappassword.Options{
 			URL:          url,
-			Insecure:     provider.Insecure,
-			TLSConfig:    tlsConfig,
+			ServerConfig: ldaputil.New(url, provider.Insecure, tlsConfig, 0), // make time limit configurable?
```

Comment:
`ldaputil.New` gives no indication of what it will return... should be `ServerConfig` or something, though that feels weird in a util package